mater/cli/src/main.rs

```diff
@@ -19,21 +19,32 @@ enum MaterCli {
         input_path: PathBuf,
 
         /// Optional path to output CARv2 file.
-        /// If no output path is given it will store the `.car` file in the same location.
+        /// If no output path is given it will store the .car file in the same location.
         output_path: Option<PathBuf>,
 
         /// If enabled, only the resulting CID will be printed.
         #[arg(short, long, action)]
         quiet: bool,
 
-        /// If enabled, the output will overwrite any existing files.
+        /// If enabled, content will be stored directly without UnixFS wrapping.
+        /// By default, content is wrapped in UnixFS format for IPFS compatibility.
         #[arg(long, action)]
-        overwrite: bool,
+        raw: bool,
+
+        /// Size of each chunk in bytes. Defaults to 256 KiB.
+        #[arg(long)]
```

Comment:
```suggestion
        #[arg(long, default_value = DEFAULT_CHUNK_SIZE)]
```

Same for `tree_width`